pkg/operators/clustermanager/bindata/bindata.go

```diff
@@ -1676,8 +1676,11 @@ rules:
   verbs: ["get", "list", "watch"]
 # Allow controller to manage placements/placementdecisions
 - apiGroups: ["cluster.open-cluster-management.io"]
-  resources: ["placements", "placementdecisions"]
+  resources: ["placements"]
   verbs: ["get", "list", "watch", "create", "update", "patch"]
```

Comment:
I think you need to remove create in placement

Reply:
'create' removed.